app/scripts/views/settings/clients.js

```diff
@@ -202,17 +200,28 @@ define(function (require, exports, module) {
       });
     },
 
+    logRefresh () {
+      this.logViewEvent('refresh');
+    },
+
     submit () {
       if (this.isPanelOpen()) {
-        this.logViewEvent('refresh');
```

Comment:
This was a bug before because `refresh` would be logged any time the view was rendered instead of on a refresh click.